src/app/register/register.component.scss

```diff
@@ -0,0 +1,120 @@
+@import "../../main-styles.scss";
```

Comment:
At work, with a recent Angular update, we had to update a bunch of our CSS files to include "min height: 1px" or something. I'll try to remember to look it up so we cover our bases there.